src/Common/tests/StaticTestGenerator/Directory.Build.props

```diff
@@ -0,0 +1,2 @@
+<!-- Standalone project. Avoid being tied to the build infrastructure in the repo. -->
+<Project/>
```

Comment:
Just as an FYI, you don't need to add this empty files if you want to avoid this imports to happen. You can add the following properties to your .csproj:

`<ImportDirectoryBuildProps>false</ImportDirectoryBuildProps>`
`<ImportDirectoryBuildTargets>false</ImportDirectoryBuildTargets>

Reply:
Oh, excellent, thanks.  I'm working on a follow-up change with additional improvements; I'll include this there.

Reply:
@safern, I just tried that and I get a bunch of warnings like:
```
warning MSB4011: "C:\Users\stoub\.nuget\packages\microsoft.sourcelink.common\1.0.0-beta2-18618-05\build\Microsoft.SourceLink.Common.props" cannot be imported again. It was already imported at "d:\repos\corefx\artifacts\toolset\Common\Tools.proj.nuget.g.props (17,5)". This is most likely a build authoring error. This subsequent import will be ignored. [d:\repos\corefx\src\Common\tests\StaticTestGenerator\StaticTestGenerator.csproj]
```

Reply:
Hm that should work. Can you point me to the change?

Reply:
Hmm that looks wrong, maybe a msbuild bug? 

Another thing you could do, which we've done in the past, is just add a `Directory.Build.props` file, and then add a `PropertyGroup` which disables the targets import, that will cause it to be cleaner by eliminating 1 empty file.

cc: @rainersigwald this should work, is there a known issue in MSBuild for this?

Reply:
I see that MSBuild considers the property, is this maybe an ordering issue?

https://github.com/Microsoft/msbuild/blob/master/src/Tasks/Microsoft.Common.props#L32

Reply:
Yes, it's an ordering issue. When using the `<Project Sdk=` form, there's an implicit import of `Sdk.props` at the very top of the project. That includes various files, including eventually `Microsoft.Common.props`, which does the `Directory.Build.props` import.

The possibilities are
* Empty d.b.props/empty d.b.targets
* d.b.props that defines `<ImportDirectoryBuildTargets>false</ImportDirectoryBuildTargets>` (and nothing else)
* Moving to the [explicit import form](https://docs.microsoft.com/en-us/visualstudio/msbuild/how-to-use-project-sdk?view=vs-2017#reference-a-project-sdk):

```xml
<Project>
 <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />

 <!-- Your actual project contents here -->

 <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
</Project>
```

The last one is unusual enough that I recommend one of the others.

Reply:
> Empty d.b.props/empty d.b.targets

😢 

I guess there is not a good way for this to be fixed, since the SDK props will always be imported first when using the implicit SDK import and because of MSBuild's evaluation, this will probably never work with it.